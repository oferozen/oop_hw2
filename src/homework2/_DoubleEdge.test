#add a double edge (two edges opposite direction)

# Create a graph
CreateGraph graph1

# Add a pair of nodes to graph1
CreateNode n1 10
CreateNode n2 3
AddNode graph1 n1
AddNode graph1 n2

# Add the directed edge (n1,n2)
AddEdge graph1 n1 n2

# Graph status
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2

# Add the directed edge (n2,n1)
AddEdge graph1 n2 n1

# Graph status
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2