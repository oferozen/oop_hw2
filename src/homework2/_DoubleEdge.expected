#add a double edge (two edges opposite direction)

# Create a graph
created graph graph1

# Add a pair of nodes to graph1
created node n1 with cost 10
created node n2 with cost 3
added node n1 to graph1
added node n2 to graph1

# Add the directed edge (n1,n2)
added edge from n1 to n2 in graph1

# Graph status
graph1 contains: n1 n2
the children of n1 in graph1 are: n2
the children of n2 in graph1 are:

# Add the directed edge (n2,n1)
added edge from n2 to n1 in graph1

# Graph status
graph1 contains: n1 n2
the children of n1 in graph1 are: n2
the children of n2 in graph1 are: n1
